front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftSections/HomepageBanner/index.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+
+// components
+import { Box, Toggle, Button, colors } from '@citizenlab/cl2-component-library';
+import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
+
+// craft
+import { useNode } from '@craftjs/core';
+
+// hooks
+import SignedOutHeader from 'containers/HomePage/SignedOutHeader';
+
+import messages from '../../../messages';
+import SignedInHeader from 'containers/HomePage/SignedInHeader';
+import { useSearchParams } from 'react-router-dom';
+import {
+  CTASignedInType,
+  CTASignedOutType,
+  THomepageBannerLayout,
+} from 'api/home_page/types';
+import { ImageSizes, Multiloc } from 'typings';
+
+export interface IHomepageSettingsAttributes {
+  banner_layout: THomepageBannerLayout;
+  banner_signed_in_header_multiloc: Multiloc;
+  banner_signed_out_header_multiloc: Multiloc;
+  banner_signed_out_subheader_multiloc: Multiloc;
+  banner_signed_out_header_overlay_color: string | null;
+  // Number between 0 and 100, inclusive
+  banner_signed_out_header_overlay_opacity: number | null;
+  header_bg: ImageSizes | null;
+  banner_cta_signed_in_text_multiloc: Multiloc;
+  banner_cta_signed_in_type: CTASignedInType;
+  banner_cta_signed_in_url: string | null;
+  // cta_signed_out
+  banner_cta_signed_out_text_multiloc: Multiloc;
+  banner_cta_signed_out_type: CTASignedOutType;
+  banner_cta_signed_out_url: string | null;
+  banner_avatars_enabled: boolean;
+}
+
+type Props = {
+  homepageSettings: IHomepageSettingsAttributes;
+};
+
+const HomepageBanner = ({ homepageSettings }: Props) => {
+  const [search] = useSearchParams();
+
+  return search.get('variant') === 'signedIn' ? (
```

Comment:
Why not rely on the `authUser` to decide whether to show the header or not? Or is this something temporary?

Reply:
This is the preview of the homepage - the user is always logged in as an admin to be able to even access this page but we need to provide a way for them to preview the homepage in the way that it would look for signed out users.

Reply:
Ah, right. I guess then eventually this logic should only be reached if the user is indeed an admin / the widget is viewed in the context of the 'preview' and not the actual homepage for non-admins, right?

Reply:
Yes, this logic is only applicable for the craft component, it doesn't affect the actual home page.